x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/historical_data/has_data.spec.ts

```diff
@@ -26,7 +26,11 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     });
 
     describe('when there is data', () => {
+      let apmSynthtraceEsClient: ApmSynthtraceEsClient;
+
       before(async () => {
+        apmSynthtraceEsClient = await synthtrace.createApmSynthtraceEsClient();
```

Comment:
Remember to clean up synthtrace,
`after(async () => {
        await apmSynthtraceEsClient.clean();
      });`

Reply:
It’s already done further down in the file https://github.com/elastic/kibana/blob/43a3a65451d3b43bef11575b1a850d56c64b64a4/x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/historical_data/has_data.spec.ts#L55